virtool/groups/data.py

```diff
@@ -49,10 +50,17 @@ async def create(self, group_id: str) -> VTGroup:
         :return: the group
         """
         try:
+            user_list = []
+            async for user in self._db.users.find():
+                if group_id in user["groups"]:
+                    user_minimal = get_user_minimal(user)
+                    user_list.append(user_minimal)
```

Comment:
Use `$in` MongoDB operator.

https://www.mongodb.com/docs/manual/reference/operator/query/in/